recipes/cythonbiogeme/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "cythonbiogeme" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cythonbiogeme-{{ version }}.tar.gz
+  sha256: f139342944c849b47ce5aaa548e68aa179c195de93ab76be78353f4309684d0e
+
+build:
+  number: 3
+  script: /bin/rm -f pyproject.toml && {{ PYTHON }} -m pip install . -vv  # [unix]
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

Should be able to keep pyproject.toml and this should run on windows too (at least to start to see if we need a special script.